src/_pytest/debugging.py

```diff
@@ -263,9 +263,9 @@ def _find_last_non_hidden_frame(stack):
 
 
 def post_mortem(t):
-    class Pdb(pytestPDB._pdb_cls):
+    class Pdb(pytestPDB._pdb_cls, object):
```

Comment:
Wondering, is it safe to mix base metaclasses like this in Python 2?

Reply:
Not sure what you mean with metaclasses.. the inheritance from `object` is required here to make it a new-style class.. `pdb.Pdb` does not have it.

Reply:
> pdb.Pdb does not have it.

Yes, which makes it an old-style class... well if it works then it works I guess.